src/Controller/WopiController.php

```diff
@@ -250,10 +261,12 @@ protected function createNewFileEntity(FileInterface $file, string $new_file_con
       FileExists::Rename,
     );
 
-    /** @var \Drupal\file\FileInterface|null $new_file */
+    /** @var \Drupal\file\FileInterface $new_file */
     $new_file = $this->entityTypeManager->getStorage('file')->create([
       'uri' => $new_file_uri,
     ]);
+    // Parameter and return docs on entity owner methods are inconsistent.
+    // @phpstan-ignore argument.type
     $new_file->setOwnerId($file->getOwnerId());
```

Comment:
The more accurate explanation:\
The param doc on `->setOwnerId()` does not accept NULL, but practically it is possible to pass NULL to unset the owner id.\
In this context we don't care if it is NULL, we just want it to be the same as for the old file.